portfolio/src/main/webapp/index.html

```diff
@@ -11,6 +11,9 @@
 
     <link rel="stylesheet" href="./css/master.css">
 
+    <!-- FontAwesome Icons -->
+    <script src="https://kit.fontawesome.com/64201e6bf1.js" crossorigin="anonymous"></script>
```

Comment:
is crossorigin required? Just wondering, I think anonymous is the default right but interested to know if there's a reason to specify it here.

Reply:
Admittedly I'm not sure. This was the default copy-paste for the FA CDN. Looking [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin) it seems like you're right about the value being the default but if the attribute isn't defined at all it doesn't use CORS whatsoever:
> By default (that is, when the attribute is not specified), CORS is not used at all. The "anonymous" keyword means that there will be no exchange of user credentials via cookies, client-side SSL certificates or HTTP authentication as described in the Terminology section of the CORS specification, unless it is in the same origin.

Reply:
At least with Bootstrap, which also includes crossorigin anonymous, when you remove it they complain about not being able to verify using CORS, and they don't deliver the content. Interesting. Thanks for the brief rabbithole lol.

Reply:
I was looking at the same site, sorry yeah anonymous is not the same as not using CORS. I see so some servers nowadays I suppose expect CORS, and some don't.